tests/unit/spawner/test_podman_spawner.py

```
@@ -699,6 +699,44 @@ async def test_idempotent_reuse_tls_no_host_port(self) -> None:
+    @pytest.mark.asyncio
+    async def test_spawn_with_allowed_skills_does_not_raise(self, caplog) -> None:
+        """spawn(allowed_skills=[...]) succeeds through the real ABC spawn() wrapper.
+
+        Regression test for a real gap: PodmanSpawner._do_spawn() had no
+        `allowed_skills` parameter, so AgentSpawner.spawn() (which
+        unconditionally forwards allowed_skills to _do_spawn()) would
+        raise TypeError for any caller going through the public .spawn()
+        method -- not caught by tests that call ._do_spawn() directly,
+        bypassing the ABC wrapper entirely.
+        """
+        mock_podman_client, mock_podman_module = (
```

Ruff (0.16.2)

[warning] Unpacked variable `mock_podman_client` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)